src/api.js

```diff
@@ -21,3 +21,25 @@ export const getCommentsByReviewId = (review_id) => {
     return res.data.comments;
   });
 };
+
+export const getUsers = () => {
+  return reviewsApi.get("/users").then((res) => {
+    return res.data.users;
+  });
+};
+
+export const likeReview = (review_id) => {
```

Comment:
likeReview and undoLikeReview can be the same function of patchVotes (or similar). Just had the function votes on a key of inc_votes - your backend should handle both -1 and 1.